contributed_definitions/NXidentifier.nxdl.xml

```diff
@@ -31,16 +31,9 @@
              If the service is not in the list a simple `url` may be used.
              The `url` can either be a resolving service for the `identifier`
              or a fully qualified identification in itself.
+             
+             Examples: doi, urn, hdl, purl, orcid, iso, url
         </doc>
-        <enumeration>
-            <item value="doi"/>
-            <item value="urn"/>
-            <item value="hdl"/>
-            <item value="purl"/>
-            <item value="orcid"/>
-            <item value="iso"/>
-            <item value="url"/>
-        </enumeration>
```

Comment:
Just curious: Is a fixed list of values more FAIR? Free text stuff is harder to handle and worse to filter for, right? Dont think we need this here now, but for me I always favored to give some fixed values, which the user should select from - just with the idea, that it is more FAIR.

anyway, Im fine with that.

Reply:
The point why we removed this was that this list is not exhaustive, so we cannot enforce this enumeration.